frontend/packages/core/src/navigation.tsx

```diff
@@ -8,7 +8,7 @@ import {
 } from "react-router-dom";
 import type { PartialPath, State } from "history";
 
-const UTM_PARAMS = ["utm_source", "utm_medium", "utm_campaign", "utm_term", "utm_content"];
```

Comment:
just curious, why are these 3 types being removed?

Reply:
Copy paste error, should be fixed :)